packages/docs/src/examples/va-backtop/Default.vue

```diff
@@ -5,7 +5,7 @@
       false-value="bottom"
       true-inner-label="top"
       false-inner-label="bottom"
-      label="Vertical Position:"
```

Comment:
I would argue translations are needed here.

Reply:
I left the text in this place without translation